pytest_testrail/plugin.py

```diff
@@ -100,6 +112,16 @@ def clean_test_ids(test_ids):
     return [int(re.search('(?P<test_id>[0-9]+$)', test_id).groupdict().get('test_id')) for test_id in test_ids]
 
 
+def clean_test_defects(defect_ids):
+    """
+        Clean pytest marker containing testrail testcase ids.
```

Comment:
I guess this should be `Clean pytest marker containing defect ids.`

Reply:
Yeah done that. :d